src/imageLoader/wadors/loadImage.js

```diff
@@ -23,7 +23,7 @@
 
     // TODO: load bulk data items that we might need
 
-    var mediaType = 'multipart/related; type="application/octet-stream"'; // 'image/dicom+jp2';
```

Comment:
The quotation marks were added by https://github.com/chafey/cornerstoneWADOImageLoader/commit/6a792c01dc716d44c80980edb2d2f257c04cae6b commit but it's breaking Orthanc. I tried with double and single quotes but they didn't work.